ROMFS/px4fmu_common/init.d/4250_teal

```diff
@@ -0,0 +1,179 @@
+#!nsh
+#
+# @name Teal One
+#
+# @type Quadrotor x
+# @class Copter
+#
+# @output MAIN1 motor 1
+# @output MAIN2 motor 2
+# @output MAIN3 motor 3
+# @output MAIN4 motor 4
+# @output MAIN5 feed-through of RC AUX1 channel
+# @output MAIN6 feed-through of RC AUX2 channel
+#
+# @output AUX1 feed-through of RC AUX1 channel
+# @output AUX2 feed-through of RC AUX2 channel
+# @output AUX3 feed-through of RC AUX3 channel
+# @output AUX4 feed-through of RC FLAPS channel
+#
+# @maintainer Jacob Dahl <jacob.dahl@tealdrones.com>
+# @maintainer Alex Klimaj <alex.klimaj@tealdrones.com>
+#
+
+echo "Executing 4250_teal script."
+
+set VEHICLE_TYPE mc
+set MIXER quad_x
+set MIXER_AUX none
+set PWM_OUT 1234
+
+if [ $AUTOCNF == yes ]
+then
+	# battery
+	param set BAT_CAPACITY 2750
+	param set BAT_CRIT_THR 0.15
+	param set BAT_EMERGEN_THR 0.075
+	param set BAT_LOW_THR 0.20
+	param set BAT_N_CELLS 4
+	param set BAT_R_INTERNAL 0.06
+	param set BAT_SOURCE 1
+	param set BAT_V_CHARGED 4.15
+	param set BAT_V_DIV 11.1625
+	param set BAT_V_EMPTY 3.65
+	param set BAT_V_OFFS_CURR -0.0045
+
+	# primary accel
+	param set CAL_ACC_PRIME 1442826 #mpu6500
+	#param set CAL_ACC_PRIME 1445386 #mpu9250
+
+	# primary gyro
+	param set CAL_GYRO_PRIME 2360330 #mpu6500
+	#param set CAL_GYRO_PRIME 2362890 #mpu9250
+
+	#primary Mag
+	#param set CAL_MAG_PRIME 265738 #mpu9250
+
+	# sensor calibration
+	param set CAL_MAG0_ROT 0
+	param set CAL_MAG_SIDES 63
+	param set SENS_BOARD_ROT 0
+	param set COM_ARM_MAG 1.5
+	param set COM_ARM_EKF_AB 0.0032
+
+	# circuit breakers
+	param set CBRK_IO_SAFETY 22027
+	param set CBRK_USB_CHK 197848
+
+	# commander
+	param set COM_DISARM_LAND 1
+	param set COM_LOW_BAT_ACT 3
+	param set COM_LOSS_LTR_T 20
+
+	# ekf2
+	param set EKF2_AID_MASK 1
+	param set EKF2_GPS_CHECK 511
+	param set EKF2_GPS_POS_X -0.04
+	param set EKF2_IMU_POS_X -0.06
+	param set EKF2_MIN_RNG 0.07
+	param set EKF2_PCOEF_XN 0.1
+	param set EKF2_PCOEF_XP -0.5
+	param set EKF2_RNG_AID 1
+	param set EKF2_RNG_A_VMAX 20.0
+	param set EKF2_RNG_NOISE 0.2
+
+	# gps
+	param set GPS_UBX_DYNMODEL 7
+
+	# geofence
+	param set GF_ACTION 1
+
+	# land detector
+	param set LNDMC_THR_RANGE 0.50
+	param set LNDMC_XY_VEL_MAX 1.0
+	param set LNDMC_ROT_MAX 50.0
+
+	# mavlink stream configuration
+	param set MAV_1_CONFIG 102
+	param set MAV_1_RATE 20000
+
+	# mc_att_control
+	param set MC_ACRO_P_MAX 360.0
+	param set MC_ACRO_R_MAX 360.0
+	param set MC_ACRO_Y_MAX 360.0
+
+	param set MC_ROLL_P 6.0
+	param set MC_ROLLRATE_P 0.055
+	param set MC_ROLLRATE_I 0.2
+	param set MC_ROLLRATE_D 0.0012
+	param set MC_ROLLRATE_MAX 180.0
+
+	param set MC_PITCHRATE_P 0.06
+	param set MC_PITCHRATE_I 0.2
+	param set MC_PITCHRATE_D 0.0012
+	param set MC_PITCHRATE_MAX 180.0
+
+	param set MC_YAW_P 1.0
+	param set MC_YAWRATE_P 0.08
+	param set MC_YAWRATE_I 0.08
+	param set MC_YAWRATE_D 0.0
+	param set MC_YAWRATE_MAX 180.0
+
+	param set MOT_SLEW_MAX 0.15
```

Comment:
What happens without this?

Reply:
We saw weird stalling issues with our ESCs and mark thought it had something to do with slew rate for the motors. I think more likely it was due to over PWMing them, I should probably test this again.

Reply:
I talked to @AlexKlimaj about this. I was wrong, we did this because the current transients we saw while full throttling looked like short circuits to our BMS.